searx/answerers/__init__.py

```diff
@@ -42,10 +42,10 @@ def ask(self, query):
         results = []
         query_parts = list(filter(None, query.query.split()))
 
-        if query_parts[0].decode('utf-8') not in answerers_by_keywords:
+        if query_parts[0] not in answerers_by_keywords:
```

Comment:
You should probably learn the basics of encoding and their differences between Python 2 and 3.

Reply:
enlighten me